doc/run_with_mysql.md

```diff
@@ -22,8 +22,8 @@ mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0
 ```
 Using the `datasource`, you may launch an all-in-one Docker container by running:  
 ```bash
-> docker run --rm -p 8888:8888 sqlflow/sqlflow bash -c \
-"sqlflowserver --datasource='mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0' &
+> docker run --rm -p 8888:8888 -p 50051:50051 sqlflow/sqlflow bash -c \
```

Comment:
I don't think we need to expose the port 50051, since it is only needed **inside** the container.

```
  Host                                   Container

                       +-----------------------------------------------+
                       |                                               |
                       |                                               |
             Port 8888 |               localhost:50051                 |
Browser <---------------->  Jupyter <-----------------> SQLFlow server |
                       |                                               |
                       |                                               |
                       +-----------------------------------------------+
```